gk/main.c

```diff
@@ -1771,6 +1791,26 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 				ret, fe->state);
 	}
 
+	if (num_reqs > 0) {
+		uint64_t acc_size_request[num_reqs + 1];
+
+		acc_size_request[0] = 0;
+		for (i = 1; i <= num_reqs; i++) {
+			acc_size_request[i] = acc_size_request[i - 1] +
+				rte_pktmbuf_pkt_len(req_bufs[i - 1]);
+		}
+
+		ret = gk_solicitor_enqueue_bulk(gk_conf->sol_conf, req_bufs,
+			req_prio, num_reqs);
```

Comment:
1. Replace the line above with `ret = RTE_MAX(gk_solicitor_enqueue_bulk(...), 0);`.
2. Replace the repeated expression `RTE_MAX(ret, 0)` below with `ret`.

Reply:
Done.